meta-ostro/conf/distro/include/ostro-os-development.inc

```diff
@@ -15,3 +15,16 @@ OSTRO_EXTRA_MOTD () {
 # Everything ready, avoid initial sanity check.
 OSTRO_IMAGE_BUILD_MODE_SELECTED = "1"
 
+# Typical cases for normal users are to produce uncompressed images
+# either for the real target or for the virtual machine. Or both.
+# Note: the variable below uses a hack, which causes the parameter to
+# contain the string "VM" even when it refers to real machines.
+
+# Image for UEFI devices.
+# OSTRO_VM_IMAGE_TYPES = " dsk"
+
+# Image for VirtualBox.
+OSTRO_VM_IMAGE_TYPES = " dsk.vdi"
+
+# Both images.
+# OSTRO_VM_IMAGE_TYPES = " dsk dsk.vdi"
```

Comment:
The purpose of ostro-os-development.inc and ostro-os-production.inc was to switch between production and developer images, i.e. these files control the content of the images. The format of the images is orthogonal to that and thus should not be changed by these files.

My expectation was that ostroproject-ci.inc add those config changes that need to be set differently in the CI compared to what is the default when building straight after "oe-init-build-env". One such change is "require ostro-os-development.inc", the other is enabling the compressed image formats.

In addition, the default value of OSTRO_VM_IMAGE_TYPES in ostro-image.bb needs to be changed such that compression is not listed.